src/ComponentSummary.jsx

```diff
@@ -245,14 +245,10 @@ return (
       </ButtonLink>
 
       <ButtonLink href={`/edit/${src}`}>
-        {context.accountId === accountId ? (
+        {context.accountId === accountId && (
           <>
             <i className="bi bi-pencil-fill"></i> Edit
           </>
-        ) : (
-          <>
-            <i className="bi bi-git"></i> Fork
-          </>
         )}
```

Comment:
After this change there is a button without text on it:
 
<img width="560" alt="Знімок екрана 2024-02-22 о 14 35 27" src="https://github.com/near/near-discovery-components/assets/34593263/10d1148a-693f-42c8-b8be-2154ffe2cd39">

What is the expected behavior here?

Reply:
@shelegdmitriy thanks for finding this.  I've fixed this with the latest commit. I would've preferred to refactor this component into separate components for the distinct use cases of this component details page versus the Component Card in the develop navigation menu, but instead of that I've changed the card logic to ensure that the Component Card in the develop menu shows the Edit button only when viewing a component that you own, and otherwise shows only one button View Source.

Reply:
@charleslavon I totally agree with eventually breaking out a new/separate component for the `Current Component` in the main nav dropdown. These two use cases have diverged quite a bit over time and will probably continue to diverge further. Whenever this refactor happens, take a look at `navigation/CurrentComponent.tsx` in our gateway. There's some super hacky CSS in that component that would be great to rip out.